pom.xml

```diff
@@ -30,13 +30,25 @@
     </developers>
 
     <properties>
-        <jenkins.version>1.580.1</jenkins.version>
+        <jenkins.version>1.596.1</jenkins.version>
```

Comment:
I'm not sure that getActiveInstance() is a good justification for the version bump. You can just implement a Util method in your plugin and restrict this Util API